src/main/java/ui/components/FilterTextField.java

```diff
@@ -48,13 +53,15 @@ public FilterTextField(String initialText) {
     private void setup() {
         setPrefColumnCount(30);
         validationSupport.registerValidator(this, (c, newValue) -> {
-            boolean wasError = false;
+            boolean isValidFilter = true;
```

Comment:
Understood. Yup, better to revert back. In the current state it looks like an unnecessary change.